C/dp/floyd_warshal.c

```diff
@@ -0,0 +1,92 @@
+#include<stdio.h>
+int i, j, k,n,arr[10][10];
+//Floyd function 
+void floyd_Warshell ()
```

Comment:
@yashaswibiyahut Update `void floyd_Warshell ()` to `void floyd_warshall()`.

Reply:
Done @iamrajiv kindly review it.